lua/metals.lua

```diff
@@ -92,7 +92,10 @@ M.copy_worksheet_output = function()
         -- return a warning with logMessage, so we can skip it here.
       end
     end
-    execute_command({ command = "metals.copy-worksheet-output", arguments = uri }, copy_response)
+    execute_command({
+      command_params = { command = "metals.copy-worksheet-output", arguments = uri },
+      callback = copy_response,
+    })
```

Comment:
Hey! So named params like this won't actually work. They work inside of the actual parameter argument since it's a table, so it expects that format. So this will have to be reverted.

Reply:
I found it here: https://www.lua.org/pil/5.3.html
Looks `lint` or `format` commands added extra () around {} which was not my intent, because now it is indeed 1 map parameter.
I'll rollback changes

Reply:
Done

Reply:
Yea, pros and cons here basically. I miss being able to use the literal call syntax, but I also like opinionated auto formatting. So that wins for me here. There was actually a discussion about this a while back if you're curious https://github.com/JohnnyMorganz/StyLua/issues/65